README.md

```diff
@@ -77,12 +77,11 @@ Future features:
 
 
 ## ♻️ Usage
-Currently, we support 2 module formats on [NPM](https://npmjs.org/package/bpmn-visualization):
+We support various module formats on [NPM](https://npmjs.org/package/bpmn-visualization):
 - [IIFE](https://developer.mozilla.org/en-US/docs/Glossary/IIFE)
 - [ESM](https://dev.to/iggredible/what-the-heck-are-cjs-amd-umd-and-esm-ikm)
+- CommonJS
   
-⚠️ CommonJS module should come [later](https://github.com/process-analytics/bpmn-visualization-js/issues/595) ⚠️
-
 
 ### IIFE
```

Comment:
A CommonJS section to explain how use the librairie with this packaging is missing 🙁

Reply:
As far as we don't fix #478, I don't want to add more 'project' examples as it implies custom rollup/webpack tricks to ensure mxgraph configuration and lib are loaded at first.
Once this is done, we will be able to add lot of project examples (like https://github.com/process-analytics/bpmn-visualization-examples/issues/20, https://github.com/process-analytics/bpmn-visualization-examples/issues/68 - this one currently suggest to use the `commonjs` bundle)

In addition, having commonjs usage in the README doesn't fit #864

Reply:
Ok, so we need to add a comment on one of these issues or create a new one, in order not to forget to update this readme ^^

Reply:
What I wanted to say in my previous comment is 'we don't want to talk about commonjs/esm in the readme, this is only a quickstart', see #864

Reply:
In my opinion, we need to explain here or in the example repo (with a link here), the basis to use the different bundle for the user of the library.

If he doesn't know the difference (even if he knows) between the packaging, it can be complicated to use the library.

Reply:
:question: Whatever the decision is, can we defer it to #864 as we will rework the whole getting-started part of the README at that time?

Reply:
👍